CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Detect live streams and time shift availability when configuring dependent components (`PlaybackTimeLabel`, `PlaybackToggleButton`, `SeekBar`) to adjust their mode independently from the player state changes
 
 ### Fixed
+- Fix option panel closing when an option list is open
```

Comment:
Please call it "settings panel" and "option select box" (similar to the names of the affected components)